src/Log/Log.php

```diff
@@ -117,15 +109,49 @@ function ($key) {
     }
 
     /**
-     * Static helper for calling the logger methods
+     * Takes the context array, serializes it, and redacts sensitive data.
      *
-     * @param  string  $name
-     * @param  array  $arguments
+     * @unreleased
+     */
+    private function serializeAndRedactContext(array $context): array
```

Comment:
Interesting approach, I like it. Little extra work is needed on larger nested arrays, but the benefit is obvious. The only thing I would suggest here is to add more keywords to the redaction list. For example, CVV (card security code) comes in multiple variations. More about that [here](https://en.wikipedia.org/wiki/Card_security_code#Naming).

Reply:
@alaca I think @JasonTheAdams added only `card` because in GiveWP all credit card fields start with this prefix.

Reply:
Yup, that's exactly right. I'm trying to ride the line and not get into other languages and such. I did have the idea, however, to make a `getRedactionList()` method with a filter so customers can add more if they need to.

Reply:
Added a filterable redaction list in 6a04ca0